packages/design-system/src/components/Pagination/Ellipses.tsx

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 
-export default function Ellipses(): React.ReactElement {
+const Ellipses : React.FC<{}> = props => {
```

Comment:
Seems odd that we'd have to define an unused `props` parameter in order to please the docs generator. It doesn't work with the type annotation alone and using `() => {`?